v2/component_analyses.yaml

```diff
@@ -0,0 +1,288 @@
+          type: array
+          items:
+            type: string
+        libio_latest_release:
+          type: array
+          items:
+            type: string
+        gh_prs_last_year_opened:
+          type: array
+          items:
+            type: integer
+        libio_latest_version:
+          type: array
+          items:
+            type: string
+        last_updated:
+          type: array
+          items:
+            type: number
+            format: float
+        gh_subscribers_count:
+          type: array
+          items:
+            type: number
+        gh_stargazers:
+          type: array
+          items:
+            type: number
+        libio_dependents_projects:
+          type: array
+          items:
+            type: string
+        gh_forks:
+          type: array
+          items:
+            type: integer
+        libio_usedby:
+          type: array
+          items:
+            type: string
+        gh_issues_last_month_closed:
+          type: array
+          items:
+            type: integer
+        ecosystem:
+          type: array
+          items:
+            type: string
+        gh_prs_last_year_closed:
+          type: array
+          items:
+            type: integer
+        gh_issues_last_year_opened:
+          type: array
+          items:
+            type: integer
+        gh_prs_last_month_closed:
+          type: array
+          items:
+            type: integer
+        gh_prs_last_month_opened:
+          type: array
+          items:
+            type: integer
+        vertex_label:
+          type: array
+          items:
+            type: string
+        gh_contributors_count:
+          type: array
+          items:
+            type: integer
+        gh_issues_last_month_opened:
+          type: array
+          items:
+            type: integer
+        gh_issues_last_year_closed:
+          type: array
+          items:
+            type: integer
+        latest_version:
+          type: array
+          items:
+            type: integer
+        gh_open_issues_count:
+          type: array
+          items:
+            type: integer
+        name:
+          type: array
+          items:
+            type: string
+        package_relative_used:
+          type: array
+          items:
+            type: string
+        package_dependents_count:
+          type: array
+          items:
+            type: integer
+        libio_total_releases:
+          type: array
+          items:
+            type: string
+        tokens:
+          type: array
+          items:
+            type: string
+      description: Component package details
+    ComponentAnalysesDataDetails:
+      title: Component Analyses Package and Version
+      type: object
+      properties:
+        version:
+          $ref: '#/components/schemas/Version'
+        package:
+          $ref: '#/components/schemas/Package'
+      description: Component analyses data for package and version
+    CVE:
```

Comment:
what does the field indicate ? CVE == public vulnerability. Lets not use 'CVE' word to indicate public vulnerabilities

Reply:
Replaced all references to CVE